extensions/amp-a4a/rtc-documentation.md

```diff
@@ -196,6 +196,145 @@ In both cases, the requirements for an errorReportingUrl are the same:
+
+The value of `sendRegardlessOfConsentState` should either be the boolean `true` or an array of consent policy states as defined in src/consent-state.js. In a case where the RTC callout would normally be supressed (for example if the AMP-consent response is `UNKNOWN`), it will instead be sent if `sendRegardlessOfConsentState` is set to boolean `true` or an array of values that contains a match for the AMP-consent state response (e.g. `['UNKNOWN']`). If set to an array, then only the values in that array (in addition to the defualt values of `SUFFICIENT` and `UNKNOWN_NOT_REQUIRED`) are treated as valid.
+
+The setting of `sendRegardlessOfConsentState` can either be done once for the entire ad slot by setting it as a top-level attribute on the RTC Config, or be done individually by setting it on any individual callouts as needed. If it is set at both the top-level and per-callout level, then if there is a mismatch, the per-callout level "wins". This is detailed below in the examples.
+
+Here are various examples of how to do this for any given ad slot:
+
+#### Whitelist all RTC callouts for all consent states
+```html
+<amp-ad width="320" height="50"
+            type="network-foo"
+            data-slot="/1234/5678"
+            rtc-config='{
+            "vendors": {
+              "vendorA": {"SLOT_ID": "1"},
+              "vendorB": {"PAGE_ID": "2"},
+              "vendorC": {"SLOT_W": "320", "SLOT_H": "50"}
+              },
+            "urls": [
+              "https://www.AmpPublisher.biz/targetingA",
+              "https://www.AmpPublisher.biz/targetingB"
+            ],
+            "sendRegardlessOfConsentState": true}'>
+</amp-ad>
+```
+By setting `sendRegardlessOfConsentState` at top-level to `true`, this indicates that for any consent state, all of the callouts in this RTC configuration should still be sent.
+
+#### Whitelist all RTC callouts for only certain consent states
+```html
+<amp-ad width="320" height="50"
+            type="network-foo"
+            data-slot="/1234/5678"
+            rtc-config='{
+            "vendors": {
+              "vendorA": {"SLOT_ID": "1"},
+              "vendorB": {"PAGE_ID": "2"},
+              "vendorC": {"SLOT_W": "320", "SLOT_H": "50"}
+              },
+            "urls": [
+              "https://www.AmpPublisher.biz/targetingA",
+              "https://www.AmpPublisher.biz/targetingB"
+            ],
+            "sendRegardlessOfConsentState": ["UNKNOWN"]}'>
+</amp-ad>
+```
+By setting `sendRegardlessOfConsentState` to an array, this indicates that only when the page consent state matches any of the consent states in the array (in this case only `UNKNOWN`) should all of the RTC callouts still be sent. For instance, if the page state is `INSUFFICIENT`, then none of the callout specified above will be sent. If the page state is `UNKNOWN`, then all of the callouts will be sent.
```

Comment:
"in this case only UNKNOWN" in addition to the default SUFFICIENT and UNKNOWN_NOT_REQUIRED